flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
+
+export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
```

Comment:
Most other designs don't use relative paths, eg
export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_HOME)/src/$(DESIGN))/*.v))

Reply:
@maliberty Then we need to fix ASAP7 config.mk in other built-in designs.

Reply:
@vijayank88 this is true for all of the ASAP7 designs. Honestly, nangate45, tsmc65, and gf12 are all better kits to look at for the moment as an example.